doc/programming_model/data_types.md

```diff
@@ -99,7 +99,12 @@ No downconversions are allowed by default, but can be enabled using
 the floating-point math controls described in @ref
 dev_guide_attributes_fpmath_mode.
 
-
+The \f$convert_{dst\_dt}\f$ conversion is not guaranteed to be correctly
+rounded. In particular, some hardware platforms have no direct
+conversion instructions from f32 to low-precision datatypes (e.g. fp8
+or fp4), and will perform conversion through an intermediate datatype
+(e.g. f16/bf16), which may cause [double
+rounding](https://en.wikipedia.org/wiki/Rounding#Double_rounding).
 
 ### Rounding mode and denormal handling
```

Comment:
Does it make sense to place this note to [rounding mode](https://github.com/oneapi-src/oneDNN/blob/main/doc/programming_model/data_types.md#rounding-mode-and-denormal-handling) section instead?

Reply:
That is an option, but given that it is not about rounding mode but rounding accuracy, I thought it might be better to put with the general formula.

Reply:
Got it, thanks.